docs/UserGuide.md

```diff
@@ -416,22 +432,23 @@ Example: <br>
 
 * Name is a compulsory field that is case-insensitive but space-sensitive.<br>
 
-* Name must be present in Pooch Planner.<br>
+* Name must be present in PoochPlanner.<br>
 
-* Using pin command on a contact that has been pinned do not make any changes to Pooch Planner.<br>
+* Using pin command on a contact that has been pinned do not make any changes to PoochPlanner.<br>
```

Comment:
grammar errors:
1. "Using _the_ pin command..."
2.  "...~~do~~ _does_ not make any changes..."

Reply:
changed